Git.hub/Git.hub.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net8.0-windows</TargetFramework>
```

Comment:
Any specific reason?

Reply:
Like other external dependencies of GE and

>**(that allows to follow)** GitExtensions code style

GE code style is "incompatible" with .net<8